jpegenc/subblocks/quantizer/__init__.py

```diff
@@ -0,0 +1,18 @@
+
+from __future__ import absolute_import
+
+from .quantiser_core import (QuantInputStream, QuantOutputStream,
+                             QuantConfig, quantizer,)
+
+from .quantiser import (QuantIDataStream, QuantCtrl,
+                        QuantODataStream, quantizer_top,)
+
+from .divider import divider, divider_ref
+from .ramz import ramz
+
+__all__ = [
+    'QuantInputStream', 'QuantOutputStream', 'QuantConfig',
+    'quantizer', 'QuantCtrl', 'quantizer_top',
+    'QuantIDataStream', 'QuantODataStream', 'divider',
```

Comment:
Why separate interfaces (types) for the `quantizer` inputs and outputs?  They seem to be the same.  What is `QuantCtrl`?